metaserver/src/main/scala/org/langmeta/jsonrpc/Endpoint.scala

```diff
@@ -0,0 +1,30 @@
+package org.langmeta.jsonrpc
+
+import io.circe.Decoder
+import io.circe.Encoder
+import monix.eval.Task
+
+class Endpoint[A: Decoder: Encoder, B: Decoder: Encoder](val method: String) {
+  def encoderA: Encoder[A] = implicitly
+  def decoderA: Decoder[A] = implicitly
+  def encoderB: Encoder[B] = implicitly
+  def decoderB: Decoder[B] = implicitly
```

Comment:
Haven’t looked closely enough to see if this is helpful here but Encoder[A] + Decoder[A] = Codec[A]

Reply:
I couldn't find any reference to circe Codec

Reply:
My mistake, I didn't realize Circe doesn't have a Codec type.